NewHorizons/External/Modules/PropModule.cs

```diff
@@ -233,6 +233,11 @@ public class RaftInfo
             /// Position of the raft
             /// </summary>
             public MVector3 position;
+
+            /// <summary>
+            /// Acceleration of the raft
```

Comment:
I’d add the default acceleration value to this doc comment

Reply:
Ok